CelesteTAS-EverestInterop/Source/TAS/Input/Commands/MouseCommand.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Input;
+using Monocle;
+using TAS.Utils;
+
+namespace TAS.Input.Commands;
+
+public static class MouseCommand {
+    public static Point? Position { get; private set; } = null;
+
+    // "Mouse, X, Y",
+    [TasCommand("Mouse")]
+    private static void Press(string[] args) {
+        if (args.IsEmpty()) {
+            return;
+        }
+
+        if (!int.TryParse(args[0], out int x) || x > 320 || x < 0) {
```

Comment:
Surely in a window of width 320, the valid values for inside the window would either range from 0 to 319 or 1 to 320, not 0 to 320.

Reply:
Modified as appropriate.